### PR TITLE
Presubmission code cleanup sprint

### PR DESCRIPTION
**Pull request by Hugemiler**
Working on several cleanup + code sanity issues pre-submission

### CODE REVIEW

File: manuscript/TableS1.jl [hunk 1]
```diff
@@ -14,7 +14,6 @@ isdir(tablefiles()) || mkpath(tablefiles())
 
 ## 1. Loading the model files
 
-RandomForestClassifier = MLJ.@load RandomForestClassifier pkg=DecisionTree
 RandomForestRegressor = MLJ.@load RandomForestRegressor pkg=DecisionTree
 # concurrent cogScore reression from taxonomic profiles
 JLD2.@load "models/2023-02-15/regression_currentCogScores_00to06mo_onlytaxa.jld"
```
Comment (kescobo) [inline manuscript/TableS1.jl:19]: Not that all of these are getting changed in #131 (though I have not yet gone through and saved the `save` ones.

**Comment by kescobo** [comment]
@Hugemiler Where does this stand at the beginning of the week?

**Comment by Hugemiler** [comment]
Closes #126

**Comment by Hugemiler** [comment]
Addresses Issue #114 , but does not close until .tex edits (PR TBD) and supplementary tables (Issue #143 )

**Comment by Hugemiler** [comment]
Closes #120

**Pull request merged**